the-collab-lab/tcl-70-smart-shopping-list
# Added design to sign in card on homepage

## Description

This code adds a designed sign in / sign up card when users first visit the app.

## Related Issue

closes #74 

## Acceptance Criteria

- [x] There should be a card centered on the page. 
- [x] In the card there should be text that welcomes the user
- [x] In the card place the instructions to tell the user to sign in/ sign up using the button below
- [x] Redesign the button and place below instructions
- [x] Add about link that takes user to about page

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: Enhancements    |

## Updates

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/8e94d306-9196-4827-a91f-1c7309c7fb2b)



### After

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/b473c8d3-97da-4c21-90d3-4908a64b2dce)

## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin km-refactor-welcome-screen` and check that branch out with `git checkout km-refactor-welcome-screen` 
* In the terminal tun `npm start`
* In the browser, go to `http://localhost:3000/`
* Try signing in using the button
* Sign out and try navigating to the about page using the link below the button
